src/plugins/subscription-manager.py

```diff
@@ -159,6 +159,17 @@ def warnOrGiveUsageMessage(conduit):
             conduit.info(2, msg)
 
 
+def init_hook(conduit):
+    disable_system_repos = conduit.confBool('main', 'disable_system_repos', default=False)
+
+    if disable_system_repos:
+        repo_storage = conduit.getRepos()
+        for repo in repo_storage.repos.values():
+            if os.path.basename(repo.repofile) != "redhat.repo":
+                conduit.info(3, 'Disable repository %s of %s' % (repo.id, repo.repofile))
```

Comment:
I would change log level from `3` to `2`, because costumers want to see such change by default. I would also change log message to something like: `'Disabling system repository %s in file %s'`